signer/tests/integration/utxo_construction.rs

```diff
@@ -25,19 +24,19 @@ use signer::utxo::SignerBtcState;
 use signer::utxo::SignerUtxo;
 
 use crate::regtest;
+use crate::regtest::AsUtxo;
 use regtest::Recipient;
 
-use regtest::DEPOSITS_LABEL;
-use regtest::SIGNER_ADDRESS_LABEL;
-use regtest::WITHDRAWAL_LABEL;
-
-pub fn make_deposit_request(
+pub fn make_deposit_request<U>(
     depositor: &Recipient,
     amount: u64,
-    utxo: &ListUnspentResultEntry,
+    utxo: &U,
```

Comment:
Nit: Since `.clone()` is only called once I'd recommend taking ownership of `U` instead of borrowing it - leaving the decision of whether to clone or move it up to the caller.

```suggestion
    utxo: U,
```

Reply:
Oh yeah, makes sense. Will update.